_inc/client/components/module-settings/index.jsx

```diff
@@ -52,6 +89,10 @@ export let RelatedPostsSettings = React.createClass( {
 						name={ 'show_thumbnails' }
 						label={ __( 'Use a large and visually striking layout' ) }
 						{ ...this.props } />
+					<h3>Preview</h3>
```

Comment:
@oskosk **Preview** must be translatable, see [this line](https://github.com/Automattic/jetpack/blob/master/modules/related-posts/jetpack-related-posts.php#L319).